src/LanguageLine.php

```diff
@@ -27,20 +27,28 @@ public static function boot()
         });
     }
 
+    /**
+     * Get translations for group
+     *
+     * @param string $locale
+     * @param string $group
+     *
+     * @return array
+     * @see https://adamwathan.me/2016/07/14/customizing-keys-when-mapping-collections/
+     */
     public static function getTranslationsForGroup(string $locale, string $group): array
     {
         return Cache::rememberForever(static::getCacheKey($group, $locale), function () use ($group, $locale) {
-            return static::query()
+            $lines = static::query()
```

Comment:
Return statement is fixed and passes the tests, but we should have $lines ?? [] because if the translations is not found at all will get null at the end and we need array